test/base/SpecBase.scala

```diff
@@ -29,7 +31,7 @@ import play.api.test.FakeRequest
 import uk.gov.hmrc.http.HeaderCarrier
 import utils.Injector
 
-import scala.concurrent.ExecutionContext
+import scala.concurrent.{ExecutionContext, Future}
```

Comment:
Probably unused import.

What about lines 21,22? probably the same

Reply:
Good spot.